# ProcessedPoints.srv
---
Point2D[] points
